### PR TITLE
Refactor StaffController from a D on Code Climate

### PR DESCRIPTION
**Pull request by calebthompson**
https://codeclimate.com/repos/54d4fa64695680762a0024c5/StaffController

### CODE REVIEW

**Comment by calebthompson** [comment]
Looks like @mafernando already had a pretty good whack at this refactor, but I took it a little bit further as well.

**Comment by chriskacerguis** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/54d4fa64695680762a0024c5/compare/refactor-staff-controller">analyzed this pull request</a>.

**Comment by mafernando** [comment]
Looks good @calebthompson, definitely like the use of ```document_staff_params```.

File: app/controllers/staff_controller.rb [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-  param :page, :number, required: false
-  param :per_page, :number, required: false
+  param :includes, Array, in: %w(user_settings projects)
+  param :methods, Array, in: %w(gravatar allowed)
+  param :page, :number
+  param :per_page, :number
+  param :query, String, desc: 'queries against first name, last name, and email'
 
   def index
     authorize Staff
```
Comment (stackus) [inline app/controllers/staff_controller.rb]: Couldn't this line get cut now?

Comment (calebthompson) [inline app/controllers/staff_controller.rb]: Done in 5c14520

**Comment by stackus** [comment]
:+1:

**Pull request merged**